docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4544,7 +4562,23 @@ four replicas of an ARP packet. These replicas will appear in the egress
 pipeline as independent packets with egress port set to PSA device port numbers
 corresponding to SDN port numbers 5, 12, 18 and 24. For more discussion on the
 translation between SDN ports and PSA device ports, refer to the
-[PSA Metadata Translation](#sec-translation-of-port-numbers) section.
+[PSA Metadata Translation](#sec-translation-of-port-numbers) section. Each
+replica can optionally have a list of backup replicas used as fallback ports
+for multicast. When the primary port (replica) goes down, the system uses the
+first backup replica whose port is up. Whenever the primary port goes back up,
+the system will use the primary port again. In the case when the primary
+replica's ports and the backup replicas' ports are down, nothing is done for
+recovery until the primary replica's or backup replicas' ports go back up.
+When the primary and all the backups are down for a particular replica, the
+packet processing side effects can result in one of the two main expected
```

Comment:
"two main expected behaviors that the target is recommended to implement."

Nit: Can we make it clearer to explain that this is a forward reference to what you are about to say?
For example, wow about ending this sentence in a colon and then using a numbered list for the two options?